erpnext/stock/report/negative_batch_report/negative_batch_report.py

```
@@ -0,0 +1,146 @@
+		},
+	]
+
+
+def get_data(filters) -> list[list]:
+	batches = get_batches(filters)
+	companies = get_companies(filters)
+	batch_negative_data = []
+
+	for company in companies:
+		for batch in batches:
+			c, data = stock_ledger_execute(
```

Ruff (0.15.0)

[warning] Unpacked variable `c` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)